controllers/machine_controller_noderef.go

```diff
@@ -34,9 +33,8 @@ var (
 	ErrNodeNotFound = errors.New("cannot find node with matching ProviderID")
 )
 
-func (r *MachineReconciler) reconcileNodeRef(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
```

Comment:
Use `_` here, like the other context one?

Reply:
Fixed in https://github.com/kubernetes-sigs/cluster-api/pull/1603